### PR TITLE
AEROGEAR-2794 Document how to download the build artefact to device

### PR DESCRIPTION
**Pull request by vchepeli**
JIRA: [AEROGEAR-2794](https://issues.jboss.org/browse/AEROGEAR-2794)

**MOTIVATION**
Update the documentation for creating a mobile build with a section around how to access and download your artefact once it is ready using the generated QR code.

### CODE REVIEW

File: docs/walkthroughs/download-artifact-to-device.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+To download artifact you need navigate to Overview -> Mobile Clients -> <your-mobile-client>
+
+image::images/mobile-client-overview.png[openshift-overview-tab]
+
+Then go to Builds tab to open mobile client builds
+
+image::images/mobile-client-builds-tab.png[openshift-mobile-client-builds-tab]
+
+== Start Mobile Client Build
+As you see that there are not builds currently available
```
Comment (maleck13) [inline docs/walkthroughs/download-artifact-to-device.adoc]: As you see that there are not builds currently available -> As you see there are no builds currently available

Comment (odra) [reply docs/walkthroughs/download-artifact-to-device.adoc]: do we need this "Start Mobile Client Build" section? We have whole document which is dedicated  in explaining on how to start a build

File: docs/walkthroughs/download-artifact-to-device.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+You need to wait until build is completed and then press [Download] button
+
+image::images/mobile-clients-builds-build-completed.png[openshift-mobile-client-builds-build-completed]
+
+You should see bar code and use any of bar code scanner applications in Google Play for Android or
+or App Store for iPhone. If you have bar code scanner installed then you need to scan
+code and it should open URL you see on screen above
+
+[[mobile-client-manual-installation]]
+= Manual Artifact Installation
```
Comment (odra) [inline docs/walkthroughs/download-artifact-to-device.adoc]: Worth mentioning that they can also use the url that is being displayed in openshift.

File: docs/walkthroughs/download-artifact-to-device.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Then go to Builds tab to open mobile client builds
+
+image::images/mobile-client-builds-tab.png[openshift-mobile-client-builds-tab]
+
+== Start Mobile Client Build
+As you see that there are not builds currently available
+When you want to trigger new build press [Start Build] button
+
+After starting build you can see notification that build was started and also
+you see that build was triggered under BUILDS section
```
Comment (maleck13) [inline docs/walkthroughs/download-artifact-to-device.adoc]: after starting build you can see notification that build was started and also you see that build was triggered under BUILDS section -> after starting a build you can see a notification that build was started and also
you can see that build was triggered under the BUILDS section

File: docs/walkthroughs/download-artifact-to-device.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+image::images/mobile-client-builds-trigger-build.png[openshift-mobile-client-builds-trigger-build]
+
+== Download Mobile Client Artifact
+You need to wait until build is completed and then press [Download] button
```
Comment (maleck13) [inline docs/walkthroughs/download-artifact-to-device.adoc]: You need to wait until build is completed and then press [Download] button -> You need to wait until the build is completed and then press [Download] button

File: docs/walkthroughs/download-artifact-to-device.adoc [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+image::images/mobile-clients-builds-build-completed.png[openshift-mobile-client-builds-build-completed]
+
+You should see bar code and use any of bar code scanner applications in Google Play for Android or
```
Comment (maleck13) [inline docs/walkthroughs/download-artifact-to-device.adoc]: You should see a QR code and can use any QR code scanner applications in Google Play for Android

File: docs/walkthroughs/download-artifact-to-device.adoc [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+or App Store for iPhone. If you have bar code scanner installed then you need to scan
+code and it should open URL you see on screen above
```
Comment (maleck13) [inline docs/walkthroughs/download-artifact-to-device.adoc]: If you have a QR code scanner installed then you need to scan the code and it should open the download URL you see on screen above.

**Comment by vchepeli** [comment]
@maleck13, @odra updated

**Pull request merged**